Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/solar.yml

```diff
@@ -0,0 +1,15 @@
+- type: entity
+  parent: MarkerBase
+  id: RandomSolarPanelState
+  name: Random solar panel
+  components:
+  - type: Sprite
+    layers:
+      - sprite: DeltaV/Structures/Power/Generation/solar_panel.rsi
+        state: random_solar
+  - type: RandomSpawner
+    offset: 0
+    prototypes:
+      - SolarPanel
+      - SolarPanelBroken
+    rareChance: 0.65
```

Comment:
replace this with `chance:` so there's occasionally nothing. Make it like 80%-85%  or so. Rarechance is used for a separate list of prototypes is defined.